### ISSUE TITLE
Move function of PN key on non RGB to FN.

### ISSUE THREAD

**Issue by crms1496** [ChaoticConundrum/pok3r_re_firmware#3]
A feature that I have been wishing for on the non RGB Pok3r is for the PN key to be removed, as it only serves one purpose. Could the functionality of PN be moved to FN, say if the FN key was held for a few seconds?

**Comment by ChaoticConundrum** [comment]
As far as I can see, the only thing PN is used for on the POK3R is for programming keys, is that what you mean? Changing that behavior would be difficult in the existing firmware, but I will definitely keep it in mind when I get to the key programming in the new firmware. Thanks for the suggestion.

**Comment by crms1496** [comment]
Yes, it would be nice to have not have a key that is only used once in a great while. I did see your response to F1LT3R's similar request, so I am not sure if it would be possible.
